vru_use_case_scripts/plot_time_to_collision

```diff
@@ -125,6 +125,8 @@ parser.add_argument(
     required=True,
 )
 
+parser.add_argument("--plots-dir", type=Path, default=Path("plots"), required=True)
```

Comment:
Is required=True and default not a contradiction?

Reply:
Yes, that is a contradiction. I made it optional.